### PR TITLE
 Demonstrate usage of SWC loader (Updated)

### PR DESCRIPTION
**Pull request by ahangarha**
This is an up-to-date copy of #30.

I confirm the project works fine (by removing `babel_loader` when we have `webpack_loader: swc` in the config file).

Should we use this as the default setting?

- - -
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails_demo_ssr_hmr/56)

### CODE REVIEW

File: bin/webpacker
Comment (justin808) [inline bin/webpacker:1]: should these files be renamed?

Comment (ahangarha) [reply bin/webpacker:1]: Since we use Shakapacker 7, yes.

Comment (ahangarha) [reply bin/webpacker:1]: Done

**Comment by ahangarha** [comment]
@justin808 Do we need test for this?

**Review by Judahmeek — CHANGES_REQUESTED** [review]
@ahangarha please add a test or at least documentation on how to run the demo with SWC

**Review by justin808 — APPROVED** [review]

**Pull request closed without merging**